MediaBrowser.Model/System/SystemInfo.cs

```diff
@@ -137,6 +138,7 @@ public SystemInfo()
         [Obsolete("This isn't set correctly anymore")]
         public FFmpegLocation EncoderLocation { get; set; }
 
+        [Obsolete("This is no longer set")]
         public Architecture SystemArchitecture { get; set; }
```

Comment:
Set a default value / make nullable so a generated SDK doesn't fail when a later release removes the property.

Reply:
Since this field is an enum, it has a default value (32bit for this specific one iirc). How does the default value affect a generated sdk when a property is removed?

Reply:
When there is a default value it adds that value to the OpenAPI spec. An SDK will add that default value to it's source. When the server stops sending the key the SDK can fallback to that default value (or null if nullable).

Reply:
Right now (10.8.9 OpenAPI spec) there is no default value or nullability.

Reply:
Fixed